doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -969,3 +969,7 @@ Documentation
 * Moved the Wi-Fi credentials library page to the upstream :ref:`Zephyr repository <zephyr:lib_wifi_credentials>`.
 * Removed the Getting started with nRF7002 DK and Getting started with other DKs pages from the :ref:`gsg_guides` section.
   These pages were no longer relevant as the `Quick Start app`_ now also supports the nRF7002 DK.
+* Added:
```

Comment:
@de-nordic  Could you please move it as per comment I left? Wrong change was appiled

Reply:
Moved entire thing up, is that ok?